MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -64,6 +65,37 @@ def store_issue_details(issue, repo, db):
 	store_quotation_result = store_on_fs(extract_quotation_content(issue['body']), issue_content_quotation_path)
 	if store_quotation_result:
 		issue_details['issue_content_quotation_path'] = issue_content_quotation_path
+
+	content_extracted = extract_jenkins_link_and_testname(issue['body'])
+	jenkins_links = content_extracted[0]
+	test_names = content_extracted[1]
+	if issue_quoted_jenkins_links != []:
+    		issue_details['issue_quoted_jenkins_links'] = jenkins_links
+	if issue_quoted_test_names != []:
```

Comment:
issue_quoted_jenkins_links and issue_quoted_test_names need to be updated. Otherwise, we will never get into the if block.

Reply:
So sorry, it escaped my notice. I have made the changes now. @llxia requesting your review.
Thank You so much.